LondonBritishLibrary/orient/BLorient488.xml

```diff
@@ -230,34 +238,26 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
                                     </desc>
                                     <q xml:lang="gez">ነገሥት፡ ዘቅ' ፡ መ' ፡ ዓለም።</q>
-                                                                        
-                                  
                                 </item>
-                                
-                                
-                                
-                                
-                                
-                                
                             </list>
                         </additions>
                         
                         
                         
                         <bindingDesc>
                             <binding contemporary="true" xml:id="binding">
-                                
-                                <decoNote xml:id="b0">In the original boards lined with striped silk and covered with blind-tooled leather. </decoNote>
+                                <decoNote xml:id="b0">In the original boards, with textile inlays of striped silk and covered with blind-tooled leather. 
```

Comment:
Here is a style deviation. I usually would use, as above and below: Two wooden boards, or just wooden boards, as above and below. Otherwise we need to add a not on originality/non originality to many elements.

Reply:
Good, "original" is already covered by "contemporary="true"" (Wright's formulation, hence the style deviation).